general-concepts/tree/text.xml

```diff
@@ -38,6 +38,24 @@ The basic layout of the portage tree is as follows:
   <li>Eclasses directory, <c>eclass/</c></li>
   <li>Licenses directory, <c>licenses/</c></li>
   <li>
+  Metadata directory, <c>metadata/</c>. Most of the listed contents are not
+  included in the main Git tree, but are generated by the
+  <uri link="::general-concepts/git-to-rsync">Git to Rsync</uri> process or
+  included there from other repositories.
```

Comment:
The inclusion of files from other repos is also a part of the git-to-rsync process. So we should rewrite this sentence. How about the following? :

```
Metadata directory, <c>metadata/</c>. Most of the listed contents are not kept directly in the main Git tree but instead auto-generated or included from other repositories as part of the <uri link="::general-concepts/git-to-rsync">Git to Rsync</uri> process.
```